src/test/java/org/openstreetmap/atlas/checks/maproulette/MapRouletteUploadCommandTest.java

```diff
@@ -57,7 +60,39 @@ public void testCountryFilter()
     public void testExecute()
     {
         final String[] additionalArguments = {};
-        this.runAndTest(additionalArguments, 1, 2, 2);
+        this.runAndTest(additionalArguments, 1, 3, 3);
+    }
+
+    @Test
+    public void testGetCountryDisplayName()
+    {
+        final Optional<String> countries1 = Optional.of("CAN,MEX");
+        final Optional<String> countries2 = Optional.of("USA");
+        final MapRouletteUploadCommand command = new MapRouletteUploadCommand();
+        final String displayCountryNames1 = command.getCountryDisplayName(countries1);
+        final String displayCountryNames2 = command.getCountryDisplayName(countries2);
+
+        Assert.assertEquals("Canada, Mexico", displayCountryNames1);
+        Assert.assertEquals("United States", displayCountryNames2);
+    }
+
+    @Test
+    public void testGetCustomChallengeName()
+    {
+        final String[] additionalArguments = { String.format("-config=%s",
+                MapRouletteUploadCommandTest.class.getResource("checksConfig.json").getPath()) };
+
+        final TestMapRouletteConnection connection = this.run(additionalArguments);
+        final Set<Project> projects = connection.uploadedProjects();
+
+        final List<String> challengeNames = projects.stream().flatMap(project -> connection
+                .challengesForProject(project).stream().map(Challenge::getName))
+                .collect(Collectors.toList());
+        Collections.sort(challengeNames);
+
+        Assert.assertEquals("Canada - Spiky Buildings", challengeNames.get(0));
+        Assert.assertEquals("Mexico, Belize - Intersecting Lines", challengeNames.get(1));
+        Assert.assertEquals("United States - Address Point Match", challengeNames.get(2));
```

Comment:
nice testing! cool to see it java and how I can try and create testing in python.